DistributionSystemApi/DistributionSystemApi/Services/RecipientGroupService.cs

```diff
@@ -0,0 +1,156 @@
+﻿using DistributionSystemApi.Data.Entities;
+using DistributionSystemApi.Data;
+using Microsoft.EntityFrameworkCore;
+using DistributionSystemApi.Requests;
+using DistributionSystemApi.Responses;
+
+namespace DistributionSystemApi.Services
+{
+    public class RecipientGroupService
+    {
+        private readonly ContentDistributionSystemContext _context;
+
+        public RecipientGroupService(ContentDistributionSystemContext context)
```

Comment:
all services should be covered by interfaces, to make sure we are not violating SOLID principles